forms/core/Application.php

```diff
@@ -644,37 +595,40 @@ public static function generateConnectPage()
             $clients = Application::$CampaignMonitor->get_clients( $auth );
             Settings::add( 'campaign_monitor_clients', $clients );
 
-	        if ( ! empty( $clients ) ) {
-		        if ( \is_array( $clients ) ) {

```

Comment:
Is the change from \is_array as opposed to is_array is intended here?

Reply:
I, have no idea. This was already existing code so I assumed it worked?